about.html

```diff
@@ -0,0 +1,159 @@
+        <p class="about-p">"Art and Majesty"</p>
+        <h1 class="about-header">GLOBAL INTERNATIONAL ART FAIR</h1>
+        <div class="line-break hide-line"></div>
+        <article>
+            <p class="about-info">
+                An international multidiscipline project that aims to converge talent in a 
+                brand new and authentic way for the modern world with people from over 80 countries taking 
+                place in April, in Nairobi.
+            </p>
+            <p class="about-contact-us">
+                Please contact us per Email for any further questions about Global International Art Fair 2023!
+            </p>
+            <p class="about-email">
+                giaf2023@giafnairobi.org
+            </p>
+        </article>  
+    </section>
+
+    <section class="about-main">
+        <div class="program-div">
+            <h2 class="about-main-header">Art Fair <br /> Global Summit 2023 Logo</h2>
+            <div class="line-break"></div>
+        </div>
+        <P class="about-main-p">
+            The logo of Global International Art Fair 2023 was designed by the talented Mark Fenekayas.
+        </P>
+        <div class="summit-logo">
+            <img 
+                src="./img/art-exhibition-logo.jpg"
+                alt=""
+                class="summit-logo-image"
+            />
+        </div>
+    </section>
+
+    <section class="past-summits">
+        <div class="program-div">
+            <h2 class="about-main-header">See the past International Art Fairs</h2>
+            <div class="line-break"></div>
+        </div>
+        <p class="about-main-p">
+            Take a look at the last two Global Art Fairs which took place in London and in Cape Town.
+        </p>
+        <div class="summit-images">
+            <div class="summit-img-one">
+                <div class="summit-img-bg">
+                    <h3>2022</h3>
+                    <p class="summit-p-two">Global Art Fair 2022 in London</p>
+                </div>
+            </div>
+            <div class="summit-img-two">
+                <div class="summit-img-bg">
+                    <h3>2021</h3>
+                    <p class="summit-p-two">Global Art Fair 2021 in Cape Town</p>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section class="partners partners-about">
+        <div class="program-div">
+            <h2 class="program-header">Partners</h2>
+            <div class="line-break"></div>
+        </div>
+
+        <div class="sponsors">
+            <img 
+                src="./img/L'officiel-logo.jpg"
+                alt=""
+                class="sponsors-logo"
+            />
+            <img 
+                src="./img/chanel-logo-illustration-958x575.jpg"
+                class="sponsors-logo"
+            />
+            <img 
+                src="./img/Zara-Emblem-700x394.jpg"
+                class="sponsors-logo"
+            />
```

Comment:
- [ ]  Hi please as mentioned by the previous reviewer, please make sure each <img> element on the repo has an alt attribute. This is crucial for the accessibility and SEO of your webApp. Also, ill recommends giving your Images a proper alternative text (`alt`).